src/ViewModels/WhatsNewViewModel.cs

```diff
@@ -3,19 +3,25 @@
 
 using System.Collections.ObjectModel;
 using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using DevHome.Common.Services;
 using DevHome.Models;
 
 namespace DevHome.ViewModels;
 
-public class WhatsNewViewModel : ObservableObject
+public partial class WhatsNewViewModel : ObservableObject

```

Comment:
Why this now needs to be partial?

Reply:
Now that the RelayCommand is generating that method, the class needs this.